docs/deployment/build.md

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
Please add these steps to https://github.com/EnAccess/micropowermanager/blob/main/docs/installation/deploy-production.md#kubernetes